### PR TITLE
Fill in the answers

### PR DESCRIPTION
**Pull request by gabyang**
Write something

### CODE REVIEW

**Comment by rldefa** [comment]
Merge to wrong branch

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (rldefa) [inline README.md]: new line?

**Review by rldefa — CHANGES_REQUESTED** [review]
Reason why i ask to merge to `submit` branch is so that I force a merge conflict, which you will have to learn to resolve.

**Comment by porckhop** [comment]
how do i restore deleted files?

**Comment by gabyang** [comment]
sorry for the clutter, I was just experimenting around and didnt realise that the timeline is being changed here

**Comment by rldefa** [comment]
Screenshot 2021-09-14 at 1 19 43 PM
Merge conflict not yet resolved properly. Can refer to my notes as to how to do it.

**Pull request closed without merging**